index.js

```diff
@@ -96,10 +96,13 @@ noble.on('stateChange', (state) => {
             setTimeout(() => {
                 log('stopping scan after timeout');
                 noble.stopScanning();
```

Comment:
Need to use `Object.keys(devices)`, as an empty object will have length > 0 anyway